src/suites/cts/validation/vertex_input.spec.ts

```diff
@@ -24,7 +24,9 @@ function clone(descriptor: GPUVertexInputDescriptor): GPUVertexInputDescriptor {
 
 class F extends ValidationTest {
   async init(): Promise<void> {
-    await Promise.all([super.init(), this.initGLSL()]);
+    // Give initGLSL the opportunity to throw SkipTestCase before super.init() pushes error scopes.
```

Comment:
Is this first because `finalize()` is not run if the test is skipped and we don't want to leave error scopes open?

If we called `finalize()` in a `finally` block as mentioned above, then we avoid this problem.